src/api/github-client.ts

```diff
@@ -0,0 +1,239 @@
+/*********************************************************************
+ * Copyright (c) 2020 Red Hat, Inc.
```

Comment:
now we're in 2021

Reply:
Yeah, thanks! I started work on that file at the end of 2020, but forgot to change later.